akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -1926,7 +1926,7 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
    *
    * '''Emits when''' upstream emits an element and configured time per each element elapsed
    *
-   * '''Backpressures when''' downstream backpressures
+   * '''Backpressures when''' downstream backpressures or the incoming rate is higher than the speed limit
```

Comment:
`the speed limit` is already described [in the same comment block](https://github.com/akka/akka/blob/b7d7316c1c63a9476d9c3e39d7a870331750bc99/akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala#L1907)

```
  /**
   * Sends elements downstream with speed limited to `elements/per`. ... ... ...
```